Steer/DigitizerWorkflow/src/SimReaderSpec.cxx

```diff
@@ -163,6 +202,7 @@ DataProcessorSpec getSimReaderSpec(SubspecRange range, const std::vector<std::st
       {"firstOrbit", VariantType::Int, 1, {"First orbit in interaction sampling"}},
       {"bcPatternFile", VariantType::String, "", {"Interacting BC pattern file (e.g. from CreateBCPattern.C)"}},
       {"simFileQED", VariantType::String, "", {"Sim (QED) input filename"}},
```

Comment:
Saying filename is misleading herr, better say prefix and give some example, e.g  ``qeddir/o2`` for the qed production in directory qeddir and files like ``qeddir/o2_Kine.root`` ...